src/main/resources/db/migration/V2024_12_13_1625__CIV_16292_notice_of_discontinue_issued_defendant.sql

```diff
@@ -0,0 +1,19 @@
+/**
+ * Add scenario
+ */
+INSERT INTO dbs.scenario (name, notifications_to_delete, notifications_to_create)
+VALUES ('Scenario.AAA6.Discontinue.NoticeOfDiscontinuanceIssued.Defendant',
+        '{}',
+        '{"Notice.AAA6.Discontinue.NoticeOfDiscontinuanceIssued.Defendant": []}');
+
+/**
+ * Add notification template
+ */
+INSERT INTO dbs.dashboard_notifications_templates ( template_name, title_En, title_Cy, description_En, description_Cy
+                                                  , notification_role)
```

Comment:
being a bit nitpicky but could we have this comma on the previous line?

Reply:
it's a script alignment

Reply:
can the comma not be put up on the previous line without error? 
or `description_Cy` be brought down to line 13?